core/daemon/src/main/java/org/opennms/netmgt/daemon/DaemonTools.java

```diff
@@ -60,7 +60,6 @@ public static void handleReloadEvent(Event e, String daemonName, Consumer<Event>
                 LOG.info("Reload successful.");
 
                 ebldr = new EventBuilder(EventConstants.RELOAD_DAEMON_CONFIG_SUCCESSFUL_UEI, daemonName);
-                ebldr.addParam(EventConstants.PARM_DAEMON_NAME, daemonName);
```

Comment:
Related?

Reply:
Not related but saw duplicate params for this event while validating this.

Reply:
Ah I see - it's still added bellow.